src/main/java/net/dv8tion/jda/api/managers/RoleManager.java

```diff
@@ -56,6 +57,8 @@
     long HOIST       = 0x8;
     /** Used to reset the mentionable field */
     long MENTIONABLE = 0x10;
+    /** Used to reset the icon field */
+    long ICON        = 0x12;
```

Comment:
```suggestion
    long ICON        = 0x20;
```
These are bit masks stored as hex.
We might want to change all of these to be bit masks built by `<<` to better represent that for future contributors.